config/locales/es.yml

```diff
@@ -54,6 +54,7 @@ es:
   description: "Descripción"
   add user: "Agregar usuario"
   add new member: "Agregar un nuevo miembro"
+  add new label: "Añadir una nueva etiqueta"
```

Comment:
Others  `add` cases are using the word `Agregar` why is that different? Would be better use the same.

Reply:
We have other cases to this Spanish translation, like: 

```
  add story: "Añadir articulo"
  add note: "Añadir nota"
  add task: 'Añadir tarea'
```

So, I was thinking that `Añadir` was being used to 'something' and `Agregar` to 'someone'. But searching I realized that both are synonymous.